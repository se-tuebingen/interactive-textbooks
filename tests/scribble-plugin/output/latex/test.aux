\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\newlabel{t:x28part_x22Questionnairex5ftestx5fmanualx22x29}{{}{1}{}{Doc-Start}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1}Single{-}Choice and Multiple Choice}{3}{section.1}\protected@file@percent }
\newlabel{t:x28part_x22Singlex2dChoicex5fandx5fMultiplex5fChoicex22x29}{{1}{3}{Single{-}Choice and Multiple Choice}{section.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2}More than one Question}{4}{section.2}\protected@file@percent }
\newlabel{t:x28part_x22Morex5fthanx5fonex5fQuestionx22x29}{{2}{4}{More than one Question}{section.2}{}}
