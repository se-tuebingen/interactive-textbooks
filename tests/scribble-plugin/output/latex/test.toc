\contentsline {section}{\numberline {1}Single{-}Choice and Multiple Choice}{3}{section.1}%
\contentsline {section}{\numberline {2}More than one Question}{4}{section.2}%
