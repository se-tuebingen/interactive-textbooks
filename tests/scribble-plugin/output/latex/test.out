\BOOKMARK [1][]{section.1}{1 Single-Choice and Multiple Choice}{}% 1
\BOOKMARK [1][]{section.2}{2 More than one Question}{}% 2
